packages/extension/src/newtab/CustomLinks.tsx

```diff
@@ -25,15 +25,15 @@ export function CustomLinks({
   className,
   onLinkClick = noop,
 }: CustomLinksProps): ReactElement {
-  const { shouldUseMobileFeedLayout } = useFeedLayout();
+  const { shouldUseListFeedLayout } = useFeedLayout();
```

Comment:
You'll see these changes a lot - made from this PR because I believe `shouldUseMobileFeedLayout` is not the right term since we're using it for tablet as well.